src/clusterfuzz/stacktraces/__init__.py

```diff
@@ -366,11 +367,6 @@ def filter_crash_parameters(self, state):
     ]:
       state.crash_state = self.fuzz_target
 
-    # Add a trailing \n if it does not exist in crash state.
-    if (state.crash_state and state.crash_state != 'NULL' and
```

Comment:
Hmmm...but I wonder what cases were previously covered that no longer are.

Reply:
This code only had an effect if we replace the state either by the process name or the fuzz target. In both cases I don't see why we should append a newline. Note that for "NULL" we also (explicitly) did not do it.

Reply:
(note how ~20 lines above we already make sure that the state ends in a newline for all other cases)